src/sentry/integrations/api/endpoints/data_forwarding_index.py

```diff
@@ -0,0 +1,97 @@
+from django.db import IntegrityError, router, transaction
+from django.utils.decorators import method_decorator
+from django.views.decorators.cache import never_cache
+from drf_spectacular.utils import extend_schema
+from rest_framework import status
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationPermission
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers import serialize
+from sentry.apidocs.constants import RESPONSE_BAD_REQUEST, RESPONSE_FORBIDDEN
+from sentry.apidocs.parameters import GlobalParams
+from sentry.integrations.api.serializers.models.data_forwarder import (
+    DataForwarderSerializer as DataForwarderModelSerializer,
+)
+from sentry.integrations.api.serializers.rest_framework.data_forwarder import (
+    DataForwarderSerializer,
+)
+from sentry.integrations.models.data_forwarder import DataForwarder
+from sentry.web.decorators import set_referrer_policy
+
+
+class OrganizationDataForwardingDetailsPermission(OrganizationPermission):
+    scope_map = {
+        "GET": ["org:read"],
+        "POST": ["org:write"],
+    }
+
+
+@region_silo_endpoint
+@extend_schema(tags=["Integrations"])
+class DataForwardingIndexEndpoint(OrganizationEndpoint):
+    owner = ApiOwner.INTEGRATIONS
+    publish_status = {
+        "GET": ApiPublishStatus.PRIVATE,
+        "POST": ApiPublishStatus.PRIVATE,
```

Comment:
would we ever want to publish these in the future?

Reply:
@JoshFerge yes! I think I have it as a later milestone in the intern project :+1: